docs/api/tiler.rst

```diff
@@ -1,5 +1,48 @@
 Tiler
 ====================================
+Different logics are implemented for tile extraction in the ``tiler`` module. The constructor of the three extractors `RandomTiler <tiler.html#src.histolab.tiler.RandomTiler>`_, `GridTiler <tiler.html#src.histolab.tiler.GridTiler>`_, and `ScoreTiler <tiler.html#src.histolab.tiler.ScoreTiler>`_ share a similar interface and common parameters that define the extraction design:
+
+#. ``tile_size``: the tile size;
+#. ``level``: the extraction level, from 0 to the number of available levels; negative indexing is also possible, counting backward from the number of available levels to 0 (e.g. ``level`` =-1) means selecting the last available level);
+#. ``check_tissue``: True if a minimum percentage of tissue over the total area of the tile is required to save the tiles, False otherwise;
+#. ``tissue_percent``: number between 0.0 and 100.0 representing the minimum required ratio of tissue over the total area of the image, considered only if ``check_tissue`` equals to True (default is 80.0);
+#. ``prefix``: a prefix to be added at the beginning of the tiles' filename (optional, default is the empty string);
+#. ``suffix``: a suffix to be added to the end of the tiles' filename (optional, default is ``.png``).
+
+All ``histolab`` extractors define the ``extract`` method, used to start the extraction process on the ``Slide`` passed as parameter.
+Each extractor will automatically (i) find the largest tissue box in the WSI; (ii) generate the tiles within the tissue region; (iii) save all generated tiles or only the informative ones if the attribute ``check_tissue`` is set to True.
+
+RandomTiler
+-----------
+The `RandomTiler <tiler.html#src.histolab.tiler.RandomTiler>`_ extractor allows for the extraction of tiles picked at random within the largest tissue box. Since there is no intrinsic upper bound of the number of the tiles that could be extracted (no overlap check is performed), the number of wanted tiles must be specified.
+
+In addition to 1-6, the `RandomTiler <tiler.html#src.histolab.tiler.RandomTiler>`_ constructor requires as two additional parameters the number of tiles requested (``n_tiles``), and the random seed (``seed``), to ensure reproducibility between different runs on the same WSI. Note that less than ``n_tiles`` could be extracted from a slide with not enough tissue pixels and a lot of background, which is checked when the parameter ``check_tissue`` is set to True.
+``n_tiles`` will be interpreted as the upper bound of the number of tiles requested: it might not be possible to extract ``n_tiles`` tiles from a slide with a little tissue sample and a lot of background.
+
+The extraction procedure will (i) find the largest tissue area in the WSI; (ii) generate ``n_tiles`` random tiles; (iii) save only the tiles with enough tissue if the attribute ``check_tissue`` was set to True, save all the generated tiles otherwise.
```

Comment:
```suggestion
The extraction procedure will (i) find the regions to extract tiles from, defined by the binary mask object; (ii) generate ``n_tiles`` random tiles; (iii) save only the tiles with enough tissue if the attribute ``check_tissue`` was set to True, save all the generated tiles otherwise.
```